### DISCUSSION TITLE
Consider replacing collections OrderedDict by plain dict

### DISCUSSION THREAD

**Discussion by pombredanne** [encode/django-rest-framework#8736] [Ideas & Suggestions] [answered]
There are still many places where OrderedDict are used https://github.com/encode/django-rest-framework/search?q=ordereddict
Since the minimal version is 3.6, these could be replace by a plain dict which is now ordered: using dict is overall faster and more concise.

**Comment by realsuayip** [comment]
@carltongibson it has been a while since https://github.com/encode/django-rest-framework/pull/7524, do you think this is plausible at this moment?

**Reply by carltongibson** [reply, marked as answer]
As per the comment there, after 2.2 was EOL it's totally doable
